Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,250 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public static class Lesson4
+    {
+        /// <summary>
+        /// Indication that the user input for an integer was empty.
+        /// </summary>
+        public const int EmptyNumberInput = 0;
+
+        /// <summary>
+        /// Indication that the user input for an integer was not in range.
+        /// </summary>
+        public const int InvalidNumberInput = -1;
+
+        /// <summary>
+        /// Indication that the user input for a name string was empty.
+        /// </summary>
+        public const string EmptyName = "-";
+
+        /// <summary>
+        /// This function will prompt some user data like name and age and in the end it will print out
+        /// the users BMI.
+        /// </summary>
+        public static void Demo()
+        {
+            ProcessPerson();
+            ProcessPerson();
+        }
+
+        /// <summary>
+        /// Prints the message and requests for user input of an integer.
+        /// </summary>
+        /// <param name="message">The message to print.</param>
+        /// <returns>The parsed integer if the input was valid, 0 when empty and -1 when invalid.</returns>
+        public static int PromptInt(string message)
+        {
+            if (PromptTryParse(message, out int parsingResult, out string input))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4885761)